src/test/java/org/openmrs/contrib/qaframework/automation/Steps.java

```diff
@@ -19,7 +19,7 @@
 
 // Use english locale, of not set, the test instance should be set to english language
 public class Steps extends ReferenceApplicationTestBase {
-	
+
```

Comment:
Removed `white spaces` to obey the principle of silence for empty lines that git admires!